dev-tools/ecs-migration.yml

```diff
@@ -90,3 +90,26 @@
   to: source.domain
   alias: true
   copy_to: false
+
+# Metricbeat base fields
+- from: metricset.name
+  to: event.dataset
+  alias: true
+  copy_to: false
+
+- from: metricset.module
+  to: event.module
+  alias: true
+  copy_to: false
+
+# TODO: Is it really an alias? Or will we change the unit?
+- from: metricset.rrt
+  to: event.duration
```

Comment:
Yeah if this one is changing by a factor of 1 000, I think we can't make this an alias. We'll have to manage the deprecation a bit more manually. Example:

6.x

- Mark `metricset.rrt` as deprecated
- Keep populating `metricset.rrt` field with microseconds
- Start populating `event.duration` field with nanoseconds
- Convert all visualizations to use new field
  - Make sure to correctly set Kibana Index Pattern units for new field

7.x

- remove `metricset.rtt`